### PR TITLE
VN-354 - Allow user select mining algorithm

### PR DESCRIPTION
**Pull request by angels2it**
Language update for ticket VN-354

### CODE REVIEW

File: cs/index.json [hunk 1]
```diff
@@ -55,7 +56,11 @@
     "general" : "Obecné",
     "mining": "Těžba",
     "about" : "O programu",
-    "log_to_file" : "Protokolovat do souboru"
+    "log_to_file" : "Protokolovat do souboru",
+    "mining_algorithms": "Mining algorithms",
+    "mining_algorithms_explaination": "Select algorithm(s) that you want to mine",
+    "mining_algorithms_warning": "Please select at least one algorithm",
+    "mining_algorithms_updated": "Mining algorithms updated successfully"
   },
```
Comment (czjohns) [inline cs/index.json]:
```
"update": "Aktualizovat"
"mining_algorithms": "Těžební algoritmy",
"mining_algorithms_explaination": "Vyberte algoritmus/algoritmy pro těžbu",
"mining_algorithms_warning": "Vyberte alespoň jeden algoritmus",
"mining_algorithms_updated": "Těžební algoritmy úspěšně aktualizovány"
```

Comment (angels2it) [reply cs/index.json]: Key was changed to preferred, Please help to update it if you have time, Thanks

File: zh/index.json [hunk 2]
```diff
@@ -55,7 +56,11 @@
     "general": "通用",
     "mining": "採礦",
     "about": "關於",
-    "log_to_file": "儲存執行紀錄"
+    "log_to_file": "儲存執行紀錄",
+    "preferred_algorithms": "Preferred algorithms",
+    "preferred_algorithms_explaination": "Select algorithm(s) that you want to mine",
+    "preferred_algorithms_warning": "Please select at least one algorithm",
+    "preferred_algorithms_updated": "Preferred algorithms updated successfully"
```
Comment (pandafox) [inline zh/index.json]:
```
"update": "更新"
"preferred_algorithms": "首選的演算法",
    "preferred_algorithms_explaination": "請選擇採礦使用的演算法",
    "preferred_algorithms_warning": "請選擇至少一種演算法",
    "preferred_algorithms_updated": "首選的演算法已成功更新"
```

**Review by votuanphong1983 — APPROVED** [review]

File: cs/index.json (hunk 1 above)
Comment (czjohns) [reply cs/index.json]:
```
"preferred_algorithms": "Preferované algoritmy",
"preferred_algorithms_explaination": "Vyberte algoritmus/algoritmy pro těžbu",
"preferred_algorithms_warning": "Vyberte alespoň jeden algoritmus",
"preferred_algorithms_updated": "Preferované algoritmy úspěšně aktualizovány"
```

Comment (angels2it) [reply cs/index.json]: Thanks

**Pull request merged**